lib/toolbox/args.c

```diff
@@ -97,3 +116,75 @@ bool args_read_hex_bytes(FuriString* args, uint8_t* bytes, size_t bytes_count) {
 
     return result;
 }
+
+bool args_read_time(FuriString* args, uint32_t* value, const char* default_unit) {
```

Comment:
Looks like `time` is not a right word, `duration` maybe? It will be nice to take a look into other standard libraries and pick common term for the thing.